std_msgs/Header header

# CPU Temp
int8 temp_package
int8 temp_core_0
int8 temp_core_1

# CPU Percent
int8 cpu_percent
int8 percent_0
int8 percent_1
int8 percent_2
int8 percent_3